TelCo.ColorCoder/GetColor.cs

```diff
@@ -0,0 +1,42 @@
+using System;
+using System.Diagnostics;
```

Comment:
Naming convention for files and classes:
A general guideline is have the name of file and class it contains to be same. 
Have single class in a single file.
Better is to have separate file for each class.

File name as the function name the file contains is not correct.

Class name should be a noun and function names should be verb.
and hence File name would be noun too.